samples/cellular/modem_shell/README.rst

```diff
@@ -470,40 +470,8 @@ See :ref:`modem_trace_module` for more information on how to configure modem tra
 You need a trace backend that can store modem traces if you want to upload modem traces to the cloud.
 The flash backend can store modem traces to the external flash on the nRF91 Series DK and can be retrieved for uploading.
 
-To enable modem traces with the flash backend, use the :file:`overlay-modem-trace-flash.conf` configuration file.
-This also requires a devicetree overlay for the external flash (:file:`nrf9160dk_ext_flash.overlay` for the nRF9160 DK or :file:`nrf9161dk_ext_flash.overlay` for the nRF9161 DK, depending on the DK you are using).
-
-Send to Memfault
-----------------
-
-To register an account and obtain the project key, refer to the :ref:`using_memfault` section of the :ref:`ug_memfault` guide.
-The Memfault overlay config (:file:`overlay-memfault.conf`) includes the most common configuration options for using the Memfault with modem traces.
-
-After a modem trace session, prepare sending the trace data to Memfault using ``modem_trace send memfault``.
-This informs the `Memfault-SDK`_ about a Custom Data Recording (CDR) that will be sent as part of the next data transfer to Memfault.
-To trigger sending immediately, it's possible to use the Memfault shell command ``mflt post_chunks``.
-
-Follow these steps to download the modem trace data:
-
-   a. In a web browser, navigate to `Memfault`_.
-   #. Log in to your account and select the project you created earlier.
-   #. Navigate to :guilabel:`Fleet` > :guilabel:`Devices` in the left side menu.
-   #. Select the **device** that sent a modem trace.
-   #. Navigate to the :guilabel:`Timeline` tab.
-   #. Find the CDR in the timeline and click on it.
-   #. Select :guilabel:`Download` from the pop-up window.
-
-See the following figure, which shows how to download the modem trace data in the `Memfault`_:
-
-.. figure:: /images/modem_shell_trace_download.png
-   :alt: Modem trace download
-
-   Modem trace download
-
-.. note::
-   The conversion of modem trace file to a Wireshark-compatible format is available in the `Cellular Monitor`_ tool of the nRF Connect for Desktop.
-
-To build the MoSh sample for an nRF91 Series DK with modem trace flash backend support, see :ref:`modem_shell_trace_flash_support`.
+To enable modem traces with the flash backend, build with the ``nrf91-modem-trace-ext-flash`` snippet for an nRF91 series DK that has external flash.
```

Comment:
This comment is really about the whole "Modem traces" chapter but I cannot add it to line 462 or 465. We haven't accurately documented the commands that are coming through other Kconfig's such as `wifi`, `lwm2m`, etc. if there is good documentation elsewhere. Modem traces are an important piece and I'd probably like some statement about these. Maybe a link to modem trace shell documentation? What do you think @tokangas?

Reply:
Good point. I stripped down the doc in modem_shell and link to nrf_modem_lib docs instead for details.